.github/workflows/i18n-validation.yml

```
@@ -0,0 +1,12 @@
+name: i18n-validation
+on:
+  push:
+    paths:
+      - '**/locales/**'
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)